### ISSUE TITLE
stock_invoicing_type doesn't group incoming pickings 

### ISSUE THREAD

**Issue by anajuaristi** [odoomrp/odoomrp-wip#1008]
Hi

If module stock_invoicing_type is installed, system is always NOT GROUPING pickings, so creates as many invoices as picking number.

I think we should erase this module from repo and include in utils or even avanzosc preventing that someone could be using it.  I'm not sure but I think at this time it's causing more problems than the ones it solves.

@pedrobaeza @agaldona Opinion?

**Comment by pedrobaeza** [comment]
As far as I remember, that module is for configuring if you want to group or not, depending on your stock picking type configuration.

**Comment by anajuaristi** [comment]
Yes, it is... but standar way without this module is working pretty well now so ... I think we could avoid installing it on most of implementations

**Comment by pedrobaeza** [comment]
OK, that's was my initial intention (to not have an overlay module), but fix it on core, but I don't remember if you mention a use case that was not covered on standard.

**Comment by anajuaristi** [comment]
Problem in standar is that you must select "group" or "not group" with a checkbox on wizard.
Our module makes this check invisible and takes the date from the "group" check on invoicing type on customer form.

This way avoids the user the responsability of selecting a data that has been configured by now. The module is working in sales but it's not in purchases. 
I don't know... maybe we should repair the case and let it be on utils but as it is now it's breaking the grouping on purchases so it's not usefull.

**Comment by pedrobaeza** [comment]
We can inhibit this behaviour on purchases, and then put it on utils certainly.
